File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/ItemPlacedMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.dtos.ItemDTO;

public class ItemPlacedMessage {

	public static final String TYPE = "ItemPlacedMessage";

	private ItemDTO itemDTO;

	public ItemPlacedMessage() {
	}

	public ItemPlacedMessage(ItemDTO itemDTO) {
		this.itemDTO = itemDTO;
	}

	public ItemDTO getItemDTO() {
		return itemDTO;
	}

	public void setItemDTO(ItemDTO itemDTO) {
		this.itemDTO = itemDTO;
	}
}

File: client/Client/Network/Messages/Editor/ItemPlacedMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Messages
{
    /// Author Florian Ortmann
    public class ItemPlacedMessage
    {
        public const string TYPE = "ItemPlacedMessage";
        
        public ItemPlacedMessage()
        {
        }

        public ItemPlacedMessage(ItemDTO itemDTO)
        {
            this.itemDTO = itemDTO;
        }

        private ItemDTO _itemDTO;

        public ItemDTO itemDTO
        {
            get => _itemDTO;
            set => _itemDTO = value;
        }    
    }
}
